src/signals/incident-management/containers/DefaultTextsAdmin/components/DefaultTextsForm/index.js

```diff
@@ -1,18 +1,26 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { FormBuilder, FieldGroup, Validators } from 'react-reactive-form';
-import isEqual from 'lodash.isequal';
+import { Row, Column, Button, themeColor  } from '@datapunt/asc-ui';
 import styled from 'styled-components';
 
+import { dataListType, defaultTextsType } from 'shared/types';
+
 import FieldControlWrapper from 'signals/incident-management/components/FieldControlWrapper';
 import TextInput from 'signals/incident-management/components/TextInput';
 import TextAreaInput from 'signals/incident-management/components/TextAreaInput';
 import HiddenInput from 'signals/incident-management/components/HiddenInput';
 
 import { ChevronDown, ChevronUp } from '@datapunt/asc-assets';
-import { Button, themeColor } from '@datapunt/asc-ui';
 
-import './style.scss';
+const StyledWrapper = styled.div`
+  margin-top: 33px;
+  width: 100%;
+`;
+
+const StyledColumn = styled(Column)`
+  display: block;
```

Comment:
You shouldn't be doing this; a `Column` component has flex-box layout which is dictated by its parent which it either a `Row` or a `Column` component. Making it behave as a block-level component breaks the ability to align the column.
Looking at the elements that are rendered by the `DefaultTextsForm` component, it becomes difficult to use `Column` component and have them line out to the grid, because the flex-box alignment is broken by the use of `<div>` and other elements that do not have flex-box layout.

You can do this:
```jsx
<Column span={8}>
  <Column span={2}></Column>
  <Column span={6}></Column>
</Column>
```

But not this:
```jsx
<Column span={8}>
  <Column span={2}></Column>
  <div>
    <Column span={6}></Column>
  </div>
</Column>
```

and expect the behaviour to be the same.

You could try this:
```jsx
<Column span={8}>
  <Column span={2}></Column>
  <div>
    <Row hasMargin={false}>
      <Column span={6}></Column>
    </Row>
  </div>
</Column>
```

Reply:
gedaan